client/my-sites/pages/virtual-page/index.tsx

```diff
@@ -112,11 +115,12 @@ const VirtualPage = ( {
 			<div className="page__main">
 				<a
 					className="page__title"
-					href={ editorUrl }
-					title={ translate( 'Edit %(title)s', {
-						textOnly: true,
-						args: { title },
-					} ) }
+					href={ isAdmin ? editorUrl : site.URL }
```

Comment:
It should be `previewUrl` as this is a virtual page component, we might have another one (e.g: Posts Page?)